### PR TITLE
Update the globe when vector layers are modified

### PR DESCRIPTION
**Pull request by ac-61**
This updates the vector layer displayed on the globe whenever it is modified using any of the functions in the javascript API.

### CODE REVIEW

File: src/essence/Basics/Layers_/Layers_.js [hunk 1]
```diff
@@ -1937,6 +1945,26 @@ var L_ = {
...
+            // because if the layer is toggled off, it is not on the globe
+            if (prevState) {
+                // Temporarily set layer to on
+                L_.toggledArray[s.name] = false
+
+                // Toggle the layer so its drawn in the globe
+                await L_.toggleLayer(s)
+
+                // Reset the previous layer state
+                L_.toggledArray[s.name] === prevState
```
Comment (tariqksoliman) [inline src/essence/Basics/Layers_/Layers_.js:1964]: Can we try to get rid of this line? `L_.toggleLayer()` should already be doing this on its own terms.  

Also, if `L_.toggleLayer()` doesn't need the await, I'd get rid of it.

**Review by tariqksoliman — CHANGES_REQUESTED** [review]

File: src/essence/Basics/Layers_/Layers_.js (hunk 1 above)
Comment (ac-61) [reply src/essence/Basics/Layers_/Layers_.js]: Fixed in 5f38e6800938884468eefc5b3bb1c2f1452dee52!

File: src/essence/Basics/Layers_/Layers_.js [hunk 2]
```diff
@@ -2108,3 +2130,5 @@ function parseConfig(configData, urlOnLayers) {
 }
 
 export default L_
+
+window.L_ = L_
```
Comment (tariqksoliman) [inline src/essence/Basics/Layers_/Layers_.js:2134]: Great! But now this.

**Review by tariqksoliman — CHANGES_REQUESTED** [review]

**Review by tariqksoliman — APPROVED** [review]
LGTM!

**Pull request merged**